aws-serverless-java-container-core/pom.xml

```diff
@@ -169,7 +176,7 @@
                 <version>${dependencyCheck.version}</version>
                 <configuration>
                     <skipProvidedScope>true</skipProvidedScope>
-                    <failBuildOnCVSS>7</failBuildOnCVSS>
+                    <failBuildOnCVSS>8</failBuildOnCVSS>
```

Comment:
The issue was related to a false positive. The CVE has been updated so it's no longer required.

Reply:
Hmm just noticed https://nvd.nist.gov/vuln/detail/CVE-2023-35116 came back today. https://github.com/FasterXML/jackson-databind/issues/3972 is closed.
Let me add a suppression for it...